code/package.json

```diff
@@ -31,7 +31,7 @@
     "eslint-config-airbnb-base": "^13.2.0",
     "eslint-plugin-import": "^2.18.2",
     "husky": "^3.0.4",
-    "jest": "^24.8.0",
+    "jest": "~24.8.0",
```

Comment:
Why have you changed from minor releases to patch releases?

Reply:
CRA has a dependency on Jest `24.8.0`, there is a new minor release `24.9.0` which is incompatible and kept getting bumped up.